specification/storage/data-plane/Microsoft.FileStorage/preview/2021-04-10/file.json

```diff
@@ -6881,7 +6881,7 @@
       "x-ms-parameter-location": "method"
     },
     "RenameIgnoreReadOnly": {
-      "name": "x-ms-file-rename-ignore-read-only",
+      "name": "x-ms-file-rename-ignore-readonly",
```

Comment:
Note that this feature has not been released yet, and this parameter name was incorrect.  This is not a breaking change.